package.json

```diff
@@ -2,7 +2,7 @@
   "name": "root",
   "private": true,
   "engines": {
-    "node": ">=20.1.0 <21"
+    "node": ">=22.0.0 <23"
```

Comment:
I still feel like it's not ideal to just support node 22 now since it's not even LTS yet, what about we keep part of our fleet on node 20, and fully switch if node 22 becomes LTS in october?

Reply:
Yeah I thought we agreed several standups ago to not drop support for 20